rs/cross-chain/proposal-cli/src/main.rs

```diff
@@ -18,8 +18,7 @@ use std::io::Write;
 use std::path::{Path, PathBuf};
 
 /// A fictional versioning CLI
-#[derive(Debug, Parser)] // requires `derive` feature
-#[command(about = "CLI to make canister upgrade proposals", long_about = None)]
```

Comment:
This line seems to have been removed, which I think is a mistake.

Reply:
Oh wow! great catch! thank you @gregorydemay !

Reply:
FYI: I have reviewed all 807 files and didn't find any other cases like that.